WordPressAuthenticator/Unified Auth/View Related/Password/PasswordViewController.swift

```diff
@@ -103,6 +103,13 @@ class PasswordViewController: LoginViewController {
     }
 
     override func displayError(message: String, moveVoiceOverFocus: Bool = false) {
+        // The reason why this check is necessary is that we're calling this method
+        // with an empty error message when setting up the VC.  We don't want to track
+        // an empty error when that happens.
+        if message.count > 0 {
```

Comment:
Just a suggestion - `isEmpty` is more efficient as it doesn't have to count.

Reply:
Changed.